### ISSUE TITLE
Move DualSense rumble strength settings to profile settings

### ISSUE THREAD

**Issue by Kanuan** [Ryochan7/DS4Windows#2571]
# Old DualSense rumble emulatiom = bad

DualSense rumble was composed of the following pieces of data:

1. Flags to enable rumble emulation
2. The left (heavy) and right (light) motor strength 
3. A value that "weakens" the overral rumble strength as a whole (affects both motors at the same time

DS4Windows allows users to adjust this last part between 3 strength levels, named as "high", "medium" and "low", though there are 8 strength levels in total.

The major problem with that  was with the controller itself: the strength level was more like a "strength REGION selector". Exemple:

- High Strength level
    - 1% rumble test: Controller would rumble at 87% strength
    - 100% rumble test: Controller would rumble at 100% strength
- Medium Strength level
    - 1% rumble test: Controller would rumble at ~45% strength
    - 100% rumble test: Controller would rumble at ~60% strength
- High Strength level
    - 1% rumble test: Controller would rumble at 1% strength
    - 100% rumble test: Controller would rumble at 13% strength
- Low Strength level while testing with 100% power: Controller would rumble at 13%.

# Improved rumble on newer firmware

Now, controllers running firmware 2.24 version or newer have an improved rumble emulation mode that can be activated by setting the correct flag and fixes this issue: not only at "high" the controller actually has a 0 to 100% strength range, it feels overral better.

I've already made some tests and will make an PR to 1) Set the improved rumble flag and 2) Set the "High" strength level as default since now it's at a reasonable maximum strength level.

Sources:

- https://twitter.com/kurtmargenau/status/1395778705578860547
- https://github.com/libsdl-org/SDL/blob/main/src/joystick/hidapi/SDL_hidapi_ps5.c#L513

# Moving DualSense strength level selector to the profile editor

One major thing to consider: with the improved rumble emulation the controller can do better by itself on adjusting the overral strength level than by using DS4Windows to downscale the strength.

Example: Example: setting the controller at "High" strength level then downscaling it to 50% in DS4Windows does not feel as good than just setting the controller to "Medium" strength level at 100%

Because of this, my idea is the following:

1. Move DualSense's strength level selector to the profile editor
2. Allow all 8 strength levels
3. Make DS4Windows give priority to it instead of the % based rumble control
    - basically, force 100% then let the controller take care of downscaling things
4. Add an option to force activate % based strength control for DualSense controllers _just in case_
    - Better safe than sorry I guess

Bad paint example image:

![image](https://user-images.githubusercontent.com/24910442/188947805-797f0b81-10a0-462e-a99b-7a7aed403016.png)
